ParameterDataApi.yaml

```diff
@@ -143,60 +199,69 @@ components:
     HeaderDefinition:
       type: object
       properties: 
-        startTime:
-          description: The start time for the time series
-          type: string
-          format: date-time
-        
-        endTime: 
-          description: The end time for the time series
+        messageType:
+          type: string          
+          description: The type of the message.
+
+        version:
           type: string
-          format: date-time
+          description: The version string for the parameter data messageType.
           
-        columns:
-          $ref: '#/components/schemas/ColumnDefinition'
-        
       required:
-        - startTime
-        - endTime
+        - messageType
+        - version
+
+    SingleDataDefinition:
+      type: object
+      description: A definition of a single parameter data sample. This is loosely based upon Json Time Series, Version 1.0. (https://docs.eagle.io/en/latest/reference/index.html)
+      properties:
+
+        header:
+          $ref: '#/components/schemas/HeaderDefinition'
+          
+        columns:
+          $ref: '#/components/schemas/ColumnDefinition'          
+
+        data:
+          $ref: '#/components/schemas/DataElement'
+
+      required: 
+        - header
         - columns
+        - data
 
     TimeSeriesDataDefinition:
       type: object
       description: A definition of time series data. This is loosely based upon Json Time Series, Version 1.0. (https://docs.eagle.io/en/latest/reference/index.html)
       properties:
-        docType:
-          type: string          
-          description: The type of the document
-
-        version:
-          type: string
-          description: The version string for the time series data definition
 
         header:
           $ref: '#/components/schemas/HeaderDefinition'
+          
+        columns:
+          $ref: '#/components/schemas/ColumnDefinition'
 
         data:
           $ref: '#/components/schemas/DataDefinition'
 
       required: 
-        - docType
         - header
+        - columns
         - data
 
     ValueSpec:
       type: object
       properties:
         value:
-          description: The value of the data sample described in string form. 
+          description: The value of the data sample described in string form
           type: string
           
         quality: 
-          description: The quality score of the data sample (should be bounded 0.0 .. 100.0)
+          description: The quality score of the data sample (should be bounded 0.0 .. 1.0)
```

Comment:
Is this essentially a percentage? I am not sure of the resolution in the new internal weather message but let's be consistent.

Reply:
It's ratio.

How about "description: The quality ratio of the data sample (should be bounded 0.0 .. 1.0)"